docs/concept/tsdb.md

```diff
@@ -33,7 +33,9 @@ Notably, for data of the `Stream` type, since there are no field columns, the `f
 
 ## Block
 
-The diagram below shows the detailed fields within each block. The block is the minimal unit of tsdb, which contains several rows of data. Due to the column-based design, each block is spread over several files.
+Each block holds data with the same series ID. 
+The max size of the measure block is controlled by data volume and the number of rows. Meanwhile, the max size of the stream block is controlled by data volume.
```

Comment:
Just curious is the data volume and number of rows adjustable through config?

Reply:
I think they are not able to do for now.

Reply:
No, I do not want to expose it to end users who do not have experience configuring it. However, dedicated users or contributors can access it by reading the code.